### ISSUE TITLE
Right way to configure User model inheritance?

### ISSUE THREAD

**Issue by bepetersn** [raddevon/flask-permissions#5]

**Comment by bepetersn** [comment]
Hey, so much later, I tried to throw together an example of this without making any assumptions, and still ended up getting this error: `sqlalchemy.exc.NoForeignKeysError: Can't find any foreign key relationships between 'fp_user' and 'users'.` Here's a repo, if you find time to point me in the right direction: https://github.com/bepetersn/flask-permissions-ex

**Comment by raddevon** [comment]
@bepetersn I haven't touched this in a while, but I believe your problem is because you change the table for your user class (see https://github.com/bepetersn/flask-permissions-ex/blob/master/ex/database.py#L21). I think that's breaking the relationships the extension needs.

**Comment by bepetersn** [comment]
Hi @raddevon, thanks for your reply. I realized that my example was slightly different from yours. I not only included the `__tablename__`, but also a primary key `id`. With either one I got the same error mentioned above, but removing both fixed it. 

I didn't understand before that that the user table was being defined by you as `fp_user` and mine was inheriting from it using Joined Table Inheritance. After I figured that out, things got easier. Thanks!
